subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/listener/AdbDeviceMetrics.kt

```diff
@@ -13,117 +13,151 @@ class AdbDeviceMetrics(
 
     private val prefix = runnerPrefix.append("adb")
 
-    override fun onGetSdkPropertySuccess(attempt: Int, api: Int) {
-        statsDSender.send(CountMetric(prefix.append("get-sdk-property", "success")))
+    override fun onGetSdkPropertySuccess(attempt: Int, api: Int, durationMs: Long) {
+        statsDSender.send(TimeMetric(prefix.append("get-sdk-property", "success"), durationMs))
     }
 
-    override fun onGetSdkPropertyError(attempt: Int) {
-        statsDSender.send(CountMetric(prefix.append("get-sdk-property", "error")))
+    override fun onGetSdkPropertyError(attempt: Int, durationMs: Long) {
+        statsDSender.send(TimeMetric(prefix.append("get-sdk-property", "error"), durationMs))
```

Comment:
Seems like we need both time and count
We need to discuss our reactions and scenarious

Reply:
Discussed and decided to use count aggregation of time metric